templatestore/frontend/src/pages/templateScreen.js

```diff
@@ -284,6 +284,11 @@ class TemplateScreen extends Component {
             .catch(function(error) {
                 console.log(error);
             });
+        if (renderMode === 'html') {
+            this.setState({
+                previewType: subType
```

Comment:
what if there are multiple `htmlpart`, this comes from config

Reply:
previewType will get updated each time before one presses Render/Preview